edn/tests/tests.rs

```diff
@@ -17,9 +17,22 @@ use std::iter::FromIterator;
 use num::bigint::ToBigInt;
 use num::traits::{Zero, One};
 use ordered_float::OrderedFloat;
+use edn::keyword;
+use edn::types::Value;
 use edn::types::Value::*;
 use edn::parse::*;
 
+// Helper for making wrapped keywords with a namespace.
+fn k_ns(ns: &str, name: &str) -> Value {
```

Comment:
Maybe not worth doing the namespace bit in the parser, but instead doing it in the constructor.  Since you have to check for empty strings anyway, might as well have one place that encodes the rules.